website/docs/docs/cloud/connect-data-platform/connect-starburst-trino.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: "Connect Starburst/Trino"
-description: "Configure Starburst/Trino connection."
-sidebar_label: "Connect Starburst/Trino"
+title: "Connect Starburst or Trino"
+description: "Configure Starburst or Trino connection."
+sidebar_label: "Connect Starburst or Trino"
 ---
 
-The following are the required fields for setting up a connection with Starburst Enterprise, Starburst Galaxy, and Trino cluster. Unless specified, "cluster" will mean any of these products' clusters.     
+The following are the required fields for setting up a connection with a [Starburst Enterprise](https://docs.starburst.io/starburst-enterprise/index.html), [Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/index.html), or [Trino](https://trino.io/) cluster. Unless specified, "cluster" means any of these products' clusters.
 
 | Field | Description | Examples |
 | --- | --- | --- |
 | **Host** | The hostname of your cluster. Don't include the HTTP protocol prefix. | `mycluster.mydomain.com` |
 | **Port** | The port to connect to your cluster. By default, it's 443 for TLS enabled clusters. | `443` |
-| **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino: <br/> <ul><li>`user.name`</li><li>`user.name@mydomain.com`</li></ul><br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |
+| **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino depends on configured authentication type. <br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |
```

Comment:
Cleaned up the table a tiny bit. 

```suggestion
| **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino depends on your configured authentication method. <br/><br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |
```

Reply:
Fixed now .. just fyi .. in the Trino context we support multiple "authentication types" . but method works just fine in this context as well